.github/workflows/test.yml

```diff
@@ -107,6 +107,7 @@ jobs:
             { runner: ubuntu-24.04, os: ubuntu, flavor: asan, cc: clang, flags: -D ENABLE_ASAN_UBSAN=ON },
             { runner: ubuntu-24.04, os: ubuntu, flavor: tsan, cc: clang, flags: -D ENABLE_TSAN=ON },
             { runner: ubuntu-24.04, os: ubuntu, flavor: release, cc: gcc, flags: -D CMAKE_BUILD_TYPE=Release },
+            { runner: ubuntu-24.04-arm, os: ubuntu, flavor: arm, cc: gcc, flags: -D CMAKE_BUILD_TYPE=RelWithDebInfo },
```

Comment:
You can remove the `RelWithDebInfo` build in the `build-types` job. In fact, with this I'd say we can remove the `build-types` job entirely as MinSizeRel or Ninja Multi-Config isn't really important enough to warrant a CI job.

Reply:
Either do it in this PR or I can do it in a follow up.

Reply:
@bfredl You'd need to remove the `build-types` job from the list of required checks then.